eahub/base/static/scripts/maps/heatmap.js

```diff
@@ -8,7 +8,7 @@ function mapSetup(queryStringMap, mapDataProfiles, mapDataGroups) {
 }
 
 function mapToggle(queryStringMap, mapDataProfiles, mapDataGroups) {
-  console.log(queryStringMap)
+  console.log(mapDataProfiles)
```

Comment:
Why do we have a console.log in here at all?

Reply:
thanks, removed